pints/_log_likelihoods.py

```diff
@@ -578,57 +578,78 @@ class MultiplicativeGaussianLogLikelihood(pints.ProblemLogLikelihood):
-    of the noise on the function output, while the scalar parameter ``sigma``
-    provides a baseline level of the noise standard deviation. This model leads
-    to a log likelihood of
+    Note that the scalar parameter :math:`\eta` controls the exponential
+    dependence of the noise on the function output, while the scalar parameter
+    :math:`\sigma` provides a baseline level of the noise standard deviation.
+    This model leads to a log likelihood of
 
     .. math::
-        \log{L(\theta, \sigma, \eta | \boldsymbol{x})} =
+        \log{L(\theta, \eta , \sigma | X^{\text{obs}})} =
             -\frac{n_t}{2} \log{2 \pi}
             -\sum_{i=1}^{n_t}{\log{f(t_i, \theta)^\eta \sigma}}
             -\frac{1}{2}\sum_{i=1}^{n_t}
-                \frac{(X(t_i) - f(t_i, \theta))^2}
+                \frac{(X^{\text{obs}}_{i} - f(t_i, \theta))^2}
                 {(f(t_i, \theta)^\eta \sigma)^2}
 
-    where ``n_t`` is the number of time points in the series.
+    where :math:`n_t` is the number of time points in the series, and
+    :math:`X^{\text{obs}}_{i}` the measurement at time :math:`t_i`.
+
+    For a system with :math:`n_o` outputs, this becomes
+
+    .. math::
+        \log{L(\theta, \eta , \sigma | \boldsymbol{x})} =
+            -\frac{n_tn_o}{2} \log{2 \pi}
+            -\sum ^{n_o}_{j=1}\sum_{i=1}^{n_t}{\log{f_j(t_i, \theta)^\eta
+            \sigma _j}}
+            -\sum ^{n_o}_{j=1}\sum_{i=1}^{n_t}
+                \frac{(X^{\text{obs}}_{ij} - f_j(t_i, \theta))^2}
+                {2(f_j(t_i, \theta)^\eta \sigma _j)^2},
+
+    where :math:`n_o` is the number of outputs of the model, and
+    :math:`X^{\text{obs}}_{ij}` the measurement of output :math:`j` at
+    time point :math:`t_i`.
 
     Extends :class:`ProblemLogLikelihood`.
 
     Parameters
     ----------
     ``problem``
         A :class:`SingleOutputProblem` or :class:`MultiOutputProblem`. For a
-        single-output problem two parameters are added (``eta``, ``sigma``),
-        for a multi-output problem 2 times ``n_outputs`` parameters are added.
+        single-output problem two parameters are added (:math:`\eta`,
+        :math:`\sigma`), for a multi-output problem 2 times :math:`n_o`
+        parameters are added.
     """
 
     def __init__(self, problem):
         super(MultiplicativeGaussianLogLikelihood, self).__init__(problem)
 
         # Get number of times and number of outputs
         self._nt = len(self._times)
-        self._no = problem.n_outputs()
+        no = problem.n_outputs()
+        self._np = 2 * no  # 2 parameters added per output
 
         # Add parameters to problem
-        self._n_parameters = problem.n_parameters() + 2 * self._no
+        self._n_parameters = problem.n_parameters() + self._np
 
         # Pre-calculate the constant part of the likelihood
-        self._logn = 0.5 * self._nt * np.log(2 * np.pi)
+        self._logn = 0.5 * self._nt * no * np.log(2 * np.pi)
```

Comment:
This is still a change, so will let @rcw5890 review this!

Reply:
True. I moved `self._logn` outside the sum in the likelihood computation. So we don't sum `self._logn` `no` times but just precompute the constant factor.

Reply:
Cool! Sorry @rcw5890 I got mixed up with another PR that you were already reviewing :sweat: